providers/Providers.tsx

```diff
@@ -2,14 +2,17 @@
 
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { ChatProvider } from "./ChatProvider";
+import PrivyProvider from "./PrivyProvider";
 import WagmiProvider from "./WagmiProvider";
 
 const queryClient = new QueryClient();
 
 const Providers = ({ children }: { children: React.ReactNode }) => (
   <WagmiProvider>
     <QueryClientProvider client={queryClient}>
-      <ChatProvider>{children}</ChatProvider>
+      <ChatProvider>
+        <PrivyProvider>{children}</PrivyProvider>
```

Comment:
Could you please move `<PrivyProvider>` outside of `<ChatProvider>`?
- I have a feeling `<ChatProvider />` might need access to the context from Privy.